db_script.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO mysql.user (User,Host,authentication_string,ssl_cipher,x509_issuer,x509_subject) VALUES('demouser','localhost',PASSWORD('demopassword'),'','','');
+FLUSH PRIVILEGES;
+SELECT User, Host, authentication_string FROM mysql.user;
+
+CREATE DATABASE IF NOT EXISTS financial_database;
+USE financial_database;
+
+DROP TABLE IF EXISTS financial;
+CREATE TABLE financial ( financial_id int unsigned not null auto_increment, segment varchar(128), country varchar(128), product varchar(128), discount_band varchar(128), units_sold float, manufacturing_price int, sale_price int, gross_sales float, discounts float, sales float, cogs float, profit float, date datetime, month_number int, month_name varchar(128), year int, primary key (financial_id) );
```

Comment:
### About schema design
```month_name varchar(128)```
We would never have any english month name exceed 128 length.

```product varchar(128)```
Excel has 32,767 characters limit per cells.  These will easily lead to truncation issue if data in xlsx is change.

```manufacturing_price int, sale_price int```
Price values as Integer ???

```date datetime```  
Why date time type if data will never specify the time?

Mostly Top set 128-char limit for every `varchar` fields.  Some won't need that long,  Some might exceed.  So in point of foreseeing / scalable design, this design doesn't meet this scenario.